src/win/process.c

```diff
@@ -831,7 +831,7 @@ int make_program_env(char* env_block[], WCHAR** dst_ptr) {
  */
 static WCHAR* find_path(WCHAR *env) {
   for (; env != NULL && *env != 0; env += wcslen(env) + 1) {
-    if (wcsncmp(env, L"PATH=", 5) == 0)
+    if (_wcsicmp(env, L"PATH=", 5) == 0)
```

Comment:
`_wcsicmp()` is affected by calls to `setlocale()`.  I'd just write it out in full to avoid any confusion:
```c
if ((env[0] == L'P' || env[0] == L'p') &&
    (env[1] == L'A' || env[1] == L'a') &&
    // etc
```